src/Tools/Source/CompilerGeneratorTools/Source/VisualBasicSyntaxGenerator/RedNodes/SyntaxFactsWriter.vb

```diff
@@ -86,6 +86,10 @@ Public Class SyntaxFactsWriter
                     tokenText = tokenText.Replace("""", """""")
                 End If
 
+                If tokenText IsNot Nothing AndAlso tokenText.Contains(vbCrLf) Then
```

Comment:
```suggestion
                If tokenText IsNot Nothing Then
```

Reply:
I'd prefer to keep this as is. I don't see any advantage to normalizing when there's nothing to normalize.

Reply:
The current approach is running the `Contains` check twice. This one is an unnecessary duplicate.

Reply:
I'd still prefer to keep it as is.